modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -0,0 +1,261 @@
+          value = {this.state.formData.project}
+          required = {true}
+        />
+        </div>;
+    }
+    if (this.state.configData['edc'] === 'true') {
+      edc =
+        <div>
+        <DateElement
+          name = "edcDateTaken"
+          label = "Expected Date of Confinement"
+          minYear = "2000"
+          maxYear = "2017"
+          onUserInput = {this.setFormData}
+          value = {this.state.formData.edcDateTaken}
+          required = {true}
+        />
+        <DateElement
+          name = "edcDateTakenConfirm"
+          label = "Confirm EDC"
+          minYear = "2000"
+          maxYear = "2017"
+          onUserInput = {this.setFormData}
+          value = {this.state.formData.edcDateTakenConfirm}
+          required = {true}
+        />
+        </div>;
+    }
+    if (this.state.configData['pscidSet'] === 'true') {
+      pscid =
+        <div >
+        <TextboxElement
+          name = "pscid"
+          label = "PSCID"
+          onUserInput = {this.setFormData}
+          value = {this.state.formData.pscid}
+          required = {true}
+        />
+        </div>;
+    }
+    if (!this.state.isCreated) {
+      profile =
+        <FormElement
+          name = "newProfileForm"
+          onSubmit = {this.handleSubmit} >
+        <label className = "error" >
+        {this.state.errMessage}
+        < /label>
+        <DateElement
+          name = "dateTaken"
+          label = "Date of Birth"
+          minYear = "2000"
+          maxYear = "2017"
+          onUserInput = {this.setFormData}
+          value = {this.state.formData.dateTaken}
+          required = {true}
+        />
+        <DateElement
+          name = "dateTakenConfirm"
+          label = "Date of Birth Confirm"
+          minYear = "2000"
+          maxYear = "2017"
+          onUserInput = {this.setFormData}
+          value = {this.state.formData.dateTakenConfirm}
+          required = {true}
+        />
+        {edc}
+        <SelectElement
+          name = "sex"
+          label = "Sex"
+          options = {this.state.configData.sex}
+          onUserInput = {this.setFormData}
+          value = {this.state.formData.sex}
+          required = {true}
+        />
+        <SelectElement
+          name = "site"
+          label = "Site"
+          options = {this.state.configData.site}
+          onUserInput = {this.setFormData}
+          value = {this.state.formData.site}
+          required = {true}
+        />
+        {pscid}
+        {project}
+        <ButtonElement label = "Create" id = "button" />
+        </FormElement>;
+    } else {
+      profile =
+        <div>
+        <p>New candidate created.DCCID:{this.state.newData.candID} PSCID: {this.state.newData.pscid} </p>
+        <p><a href = {'/' + this.state.newData.candID}> Access this candidate </a></p>
+        <p><a href = "/new_profile/" > Recruit another candidate </a></p>
+        </div>;
+    }
+    return (<div><Panel>{profile}</Panel></div>);
```

Comment:
```suggestion
    return (<Panel>{profile}</Panel>);
```

Reply:
might be nice also to add a "title" prop to the Panel element so that a banner appears at the top